docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '3.0'
+
+services:
+  grafana:
+    container_name: 'grafana-redshift-datasource'
+    build:
+      context: ./.config
+      args:
+        grafana_version: ${GRAFANA_VERSION:-9.1.2}
```

Comment:
I'm not the best at docker, do you know will this run grafana 9.1.2 or will it run "latest" from the dockerfile? I suppose it doesn't make a huge difference I'm just curious how this all plugs together, and if maybe we should pin this set up to a particular version when we run `yarn server`

Reply:
I'm assuming its the second one, because I imagine that they would just use 9.1.2 otherwise, but I'm not sure.